assignmentPR/conversionF1.c

```diff
@@ -74,6 +48,28 @@ int characteristic(char numString[], int* c)
 /*___________________________________________________________________________*/
 
 
+/*___________________________________________________________________________*/
+//Converts character into an integer
+int charToInt(char* charArray)
+{
+    int finalInt = 0; // The return int
+
+    //Loop through eveey alhpa numeric char in the string and convert
+    //into corresponding int.
+    for (int i = 0; charArray[i] != '\0'; i++)
+    {
+        if(alphaValidate(charArray[i]) == 1)
+        {
+            return -1; // Char is not a integer. 
+        }
+        finalInt = finalInt*10 + charArray[i] - '0'; // - '0' removes all 0's.
+    }
+  
+    // return the final integer that has been converted from the passed string.
+    return finalInt;
+}
+/*___________________________________________________________________________*/
+
 /*___________________________________________________________________________*/
 // Returns the part of a float after decimal (significand)
 int mantissa(char numString[], int* numerator, int* denominator)
```

Comment:
You should reconsider using for loops that are effectively while loops. It would help your readability to use while loops  in cases where number of iterations in unknown.